src/lib.rs

```diff
@@ -628,6 +669,25 @@ pub fn cfg<S: Into<String>>(name: S, actions: &str) -> Result<(), String> {
     set(&mut registry, name.into(), actions)
 }
 
+/// Configure the actions for a fail point at runtime.
+///
+/// Each fail point can be configured by a callback. Process will call this callback function
+/// when it meet this fail-point.
+pub fn cfg_callback<S, F>(name: S, f: F) -> Result<(), String>
+where
+    S: Into<String>,
+    F: Fn() + Send + Sync + 'static,
+{
+    let mut registry = REGISTRY.registry.write().unwrap();
+    let p = registry
```

Comment:
Reuse `set` instead.

Reply:
It may be difficult... They are different.

Reply:
Got it.